package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "yelda-webchat",
-  "version": "1.0.50",
+  "version": "1.0.51",
```

Comment:
npm run uploadBuildToS3 command with --semver=patch argument will automatically increment the npm version, so no need to do it manually

Reply:
Yes it was from the script actually